bridges/snowbridge/pallets/outbound-queue-v2/src/benchmarking.rs

```diff
@@ -67,14 +99,48 @@ mod benchmarks {
 		Ok(())
 	}
 
-	/// Benchmark for producing commitment for a single message
+	/// Benchmark for producing commitment for a single message, used to estimate the delivery
+	/// cost. The assumption is that cost of commit a single message is even higher than the average
+	/// cost of commit all messages.
 	#[benchmark]
 	fn commit_single() -> Result<(), BenchmarkError> {
-		let leaf = <T as Config>::Hashing::hash(&[100; 1]);
-		MessageLeaves::<T>::append(leaf);
+		initialize_with_one_message::<T>();
+
+		#[block]
+		{
+			OutboundQueue::<T>::commit();
+		}
+
+		Ok(())
+	}
+
+	/// Benchmark for `on_initialize` in the worst-case
+	#[benchmark]
+	fn on_initialize_when_congested() -> Result<(), BenchmarkError> {
```

Comment:
The worst case path should be the default, so we surely don't need to specialize the naming?

```suggestion
	fn on_initialize() -> Result<(), BenchmarkError> {
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/61/commits/6e442344f592ec09c35feef0b103636b1c95b764